docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -211,10 +224,10 @@ Alternatively, navigate to your Data storage directory and enter the following c
 [#CB90061]
 === Disk Write Queue (CB90061)
 
-*Background*: Having items in the disk write queue for too long could cause backoffs. 
+*Background*: Having items in the disk write queue for too long could cause backoffs.
```

Comment:
-> applications to backoff and writes to fail

Reply:
disk write queue -> Disk Write Queue for consistency?

Reply:
this isn't from my checker, vs code just removed a blank space when I checked to remove all of mine. Should that be changed in a different branch or is it okay to be fixed in this one?

Reply:
Ah I see. For small things I'm sure it's fine, since you're editing the same file anyway.

Reply:
That's true, I'll make that change now then :)

Reply:
Fixed :)